.github/workflows/ci.yml

```diff
@@ -27,3 +27,19 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+      - name: Generate README
+        uses: docker://pandoc/core:2.18
+        with:
+          entrypoint: ./docs/generate
+
+      - name: Check generated README is up to date
+        run: |
+          if [ -n "$(git status --porcelain README.md)" ]; then
```

Comment:
We may be able to somehow run this check inside the Pandoc image above, but I think running it outside the image is okay.